app/controllers/declaration/TotalPackageQuantityController.scala

```diff
@@ -39,14 +39,14 @@ class TotalPackageQuantityController @Inject()(
   override val exportsCacheService: ExportsCacheService,
   totalPackageQuantity: total_package_quantity
 )(implicit ec: ExecutionContext)
-    extends FrontendController(mcc) with I18nSupport with ModelCacheable {
+    extends FrontendController(mcc) with I18nSupport with ModelCacheable with SubmissionErrors {
 
   private val validTypes = Seq(DeclarationType.STANDARD, DeclarationType.SUPPLEMENTARY)
 
-  def displayPage(mode: Mode) = (authorize andThen journey(validTypes)) { implicit request =>
-    val form = TotalPackageQuantity.form(request.declarationType)
-    val data = request.cacheModel.totalPackageQuantity.fold(form)(form.fill)
-    Ok(totalPackageQuantity(mode, data))
+  def displayPage(mode: Mode): Action[AnyContent] = (authorize andThen journey(validTypes)) { implicit request =>
+    val form = TotalPackageQuantity.form(request.declarationType).withSubmissionErrors()
+    val datax = request.cacheModel.totalPackageQuantity.fold(form)(form.fill)
```

Comment:
Here it looks like a typo too

Reply:
It's a more trendy form of data.  Like SpaceX!